src/sphinx.md

```diff
@@ -73,6 +76,7 @@ additional data:
 
 | Action   | Description                                | Additional data              |
 |----------|--------------------------------------------|------------------------------|
+// TODO a bit unclear why we would have a Mac when there is on define outside in packet structure.
```

Comment:
Hmm yeah. This is the MAC for the next hop, so maybe I could say "MAC for next hop" or something.

Reply:
i would find "next hop mac" good.